shared/js/event_dispatcher.js

```diff
@@ -101,6 +101,22 @@
     },
 
     /**
+     * Registers listener function to be executed only first time when event
+     * occurs.
+     * @param {string} eventName Name of the event to listen for.
+     * @param {function} handler Handler to be executed once event occurs.
+     */
+    once: function(eventName, handler) {
+      var once = (...args) => {
+        this.off(eventName, once);
+
+        handler(args);
```

Comment:
Hmm, looks like I don't need ```...``` at all (not sure why I added it for the arguments) :)  since we always have only one parameter